src/main/java/seedu/address/logic/exporter/CsvExporter.java

```diff
@@ -0,0 +1,78 @@
+package seedu.address.logic.exporter;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.util.FileUtil;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.person.Note;
+import seedu.address.model.person.Person;
+
+/**
+ * Represents a class that can export to CSV
+ */
+public class CsvExporter implements Exporter {
+    private static final String CSV_HEADERS = "Name,Phone No,Email,Address,Tags,Notes";
+
+    private final Path exportSubPath = Paths.get("bizbook.csv");
+    private final ReadOnlyUserPrefs userPrefs;
+
+    /**
+     * Constructs a {@code VcfExporter} class configured with {@code UserPrefs}
+     */
+    public CsvExporter(ReadOnlyUserPrefs userPrefs) {
+        this.userPrefs = userPrefs;
+    }
+
+    @Override
+    public void exportAddressBook(ReadOnlyAddressBook addressBook) throws IOException {
+        Path exportPath = getExportPath();
+        FileUtil.createIfMissing(exportPath);
+
+        StringJoiner sj = new StringJoiner(System.lineSeparator());
+        sj.add(CSV_HEADERS);
+
+        addressBook.getPersonList()
+                .forEach(person -> sj.add(convertToCsv(person)));
+
+        FileUtil.writeToFile(exportPath, sj + System.lineSeparator());
```

Comment:
Do you think that Exporter should be under Storage, instead of Logic? Since it is interacting and writing data to hard disk?